src/auth/oauth.ts

```diff
@@ -271,7 +272,10 @@ export interface TokenExchangeGrantRequest {
 export class OAuth extends BaseAuthAPI {
   private idTokenValidator: IDTokenValidator;
   constructor(options: AuthenticationClientOptions) {
-    super(options);
+    super({
+      ...options,
+      domain: options.useMTLS ? `${mtlsPrefix}.` + options.domain : options.domain,
```

Comment:
```suggestion
      domain: options.useMTLS ? `${mtlsPrefix}.${options.domain}` : options.domain,
```

Reply:
Thanks for suggesting this, updated https://github.com/auth0/node-auth0/pull/1002/commits/0d041f4bfe706566ec014ff518df7ed49206311e